typescript/infra/config/environments/mainnet2/multisigIsm.ts

```diff
@@ -0,0 +1,98 @@
+import { ChainMap, MultisigIsmConfig } from '@hyperlane-xyz/sdk';
+
+export const multisigIsm: ChainMap<MultisigIsmConfig> = {
+  celo: {
+    threshold: 4,
+    validators: [
+      '0x1f20274b1210046769d48174c2f0e7c25ca7d5c5',
+      '0x3bc014bafa43f93d534aed34f750997cdffcf007',
+      '0xd79d506d741fa735938f7b7847a926e34a6fe6b0',
+      '0xe4a258bc61e65914c2a477b2a8a433ab4ebdf44b',
+      '0x6aea63b0be4679c1385c26a92a3ff8aa6a8379f2', // staked
```

Comment:
What do you think about moving this to sdk/consts? It would be nice to be able to import it from `hyperlane-deploy`, otherwise we need to copy and paste :(

Reply:
Love it